tools/scoring/tests/fixtures/files/proposed/match_reporting_entities.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is used as an example of a valid building configuration with
+# the most common and important features, e.g. translations and links.
+
+# This WILL match as cloud_device_id corresponds and translation is present
```

Comment:
can we re-use the existing files or consolidate?

the path needs to be reworked to be in here: https://github.com/google/digitalbuildings/tree/master/tools/scoring/tests/samples

Reply:
The existing files could be used but in that circumstance it's much more difficult to discern what is being tested.